ShopApp/ShopApp/Controller/ProductDetailsViewController.swift

```diff
@@ -70,15 +78,18 @@ class ProductDetailsViewController: UIViewController {
             let allFieldRequired = UIAlertController(title: "Message",
                                                      message: "First you have to choose a size!",
                                                      preferredStyle: .alert)
-            allFieldRequired.addAction(UIAlertAction(title: String(localized: "Ok", comment: "Ok"), style: .cancel, handler: nil))
             self.present(allFieldRequired, animated: true, completion: nil)
+
+            // dismiss after 2 seconds
+            let when = DispatchTime.now() + 2
```

Comment:
try to set as self explanatory as possible names. like "timeOfDismissal" instead of "when"